### ISSUE TITLE
Instructors should be able to remove students from their own course

### ISSUE THREAD

**Issue by firasm** [PrairieLearn/PrairieLearn#6039]
See context in #6038

tl;dr version: we should allow instructors to remove students from a course (with appropriate modals "are you sure" and "are you really really sure" because all student data will be erased)?

There are some legitimate cases of removing students from a course

- Instructor error 1 (pointing them to the wrong link, or not updating all the templates)
- Instructor error 2 (setting up access rules incorrectly and allowing Google Auth instead of institutional ID)
- Student error (trigger-happy students enrolling in all visible public courses)

Here's a sample UI for where we could do this:

<img width="1709" alt="Screen Shot 2022-07-25 at 8 28 13 PM" src="https://user-images.githubusercontent.com/2507459/180917137-387e8012-3292-40f9-b99e-862b2e9d4fec.png">

**Comment by jonatanschroeder** [comment]
Related: #4641

**Comment by kurmasz** [comment]
Another reason: I had a student enroll with two different emails (her student email and her faculty email).  Now the gradebook has two adjacent rows for her --- one full and one empty.

**Comment by reteps** [comment]
This is now possible with #12856

**Comment by firasm** [comment]
YAY!!! 🥳🥳
